pipeline/patch_VAE.py

```diff
@@ -160,25 +255,36 @@ def process_VAE(paths):
         "*_latent_space_after.pkl": array of latent vectors (after quantization)
 
     Args:
-        paths (list): list of paths, containing:
-            0 - folder for raw data, segmentation and summarized results
-            1 - folder for supplementary data
-            2 - path to VQ-VAE model weight
-            3 - list of site names
+        summary_folder (str): folder for raw data, segmentation and 
+            summarized results
+        supp_folder (str): folder for supplementary data
+        channels (list of int): indices of channels used for VAE encoding
+        model_path (str): path to model weight
+        sites (list of str): list of site names
+        input_clamp (list of float or None): if given, the lower/upper limit 
+            of input patches
 
     """
-    # these sites should be from a single condition (C5, C4, B-wells, etc..)
-    summary_folder, supp_folder, model_path, sites = paths[0], paths[1], paths[2], paths[3]
+
     assert len(set(site[:2] for site in sites)) == 1, \
         "Sites should be from a single well/condition"
     well = sites[0][:2]
+    
+    preprocess_setting={
+        0: ("normalize", 0.4, 0.05), # Phase
+        1: ("scale", 0.05), # Retardance
+        2: ("normalize", 0.5, 0.05), # Brightfield
+    }
```

Comment:
@miaecle how did you determine these normalization parameters? Are they dataset specific? @mattersoflight @bryantChhun